db/zm_update-1.33.1.sql

```diff
@@ -0,0 +1,13 @@
+--
+-- This updates a 1.33.0 database to 1.33.1
+--
+-- Add WebSite enum to Monitor.Type
+-- Add Refresh column to Monitors table
+--
+
+ALTER TABLE `MonitorPresets` 
+CHANGE COLUMN `Type` `Type` ENUM('Local','Remote','File','Ffmpeg','Libvlc','cURL','WebSite','NVSocket') NOT NULL DEFAULT 'Local' ;
+
+ALTER TABLE `Controls` 
```

Comment:
Shouldn't this be Monitors?

Reply:
No. The Monitors table was already updated here: https://github.com/ZoneMinder/zoneminder/blob/master/db/zm_update-1.31.43.sql

Reply:
I just checked doing an zmupdate.pl  going from 1.32.3 with this branch and zm_update-1.31.43.sql doesn't seem to be relevant here as this is the output of `mysqldump -d zm` for the `Monitors` table (up to the `Type` enum)

```
CREATE TABLE `Monitors` (
  `Id` int(10) unsigned NOT NULL AUTO_INCREMENT,
  `Name` varchar(64) NOT NULL DEFAULT '',
  `ServerId` int(10) unsigned DEFAULT NULL,
  `StorageId` smallint(5) unsigned DEFAULT '0',
  `Type` enum('Local','Remote','File','Ffmpeg','Libvlc','cURL') NOT NULL DEFAULT 'Local',
```

Reply:
Yeah the current zm_create.sql doesn't have WebSite in the enum.  So we do need it in this update.
While you are at it, can you add the NVSocket type as well?

Reply:
I set all three Enum's in both the create and update scripts to be: `'Local','Remote','File','Ffmpeg','Libvlc','cURL','WebSite','NVSocket'`